chalice/cli/__init__.py

```diff
@@ -367,6 +372,30 @@ def generate_sdk(ctx, sdk_type, stage, outdir):
         raise click.Abort()
 
 
+@cli.command('generate-models')
+@click.option('--stage', default=DEFAULT_STAGE_NAME,
+              help="Chalice Stage for which to generate models.")
+@click.pass_context
+def generate_models(ctx, stage):
+    # type: (click.Context, str) -> None
+    factory = ctx.obj['factory']  # type: CLIFactory
+    config = factory.create_config_obj(stage)
+    graph_builder = ApplicationGraphBuilder()
```

Comment:
Let's remove all the app graph builder stuff.  To check for restapis we can just check `config.chalice_app.routes` which is what the app graph builder does.